setup.py

```diff
@@ -139,6 +139,16 @@ def run_tests(self):
     author=meta['author'],
     author_email=meta['contact'],
     url=meta['homepage'],
+    project_urls={
+        'Documentation': (
+            'https://django-celery-results.readthedocs.io/en/latest/'
+        ),
+        'Changelog': (
+            'https://django-celery-results.readthedocs.io/en/latest/'
+            'changelog.html'
+        ),
+        'Repository': 'https://github.com/celery/django-celery-results',
```

Comment:
I wasn't sure whether I should keep these here, or define them in `__init__.py` instead as other metadata is? E.g.

```python
__documentation_url__ = (
    'https://django-celery-results.readthedocs.io/en/latest/'
)
__changelog_url__ = (
    'https://django-celery-results.readthedocs.io/en/latest/changelog.html'
)
```